From a04fbe7f3f693754d9c3f52152aa2644f4d7a0fa Mon Sep 17 00:00:00 2001
From: Shawn Lin <shawn.lin@rock-chips.com>
Date: Mon, 12 Nov 2018 17:34:42 +0800
Subject: [PATCH] phy: rockchip-inno-combphy: Remove releasing rst grant from
 PCIe init

The PCIe controller driver will take over to control this, as it need
more action after link up.

Change-Id: Ie9a690d1addf48855d385162d0d8ba4d0054c544
Signed-off-by: Shawn Lin <shawn.lin@rock-chips.com>
---
 drivers/phy/rockchip/phy-rockchip-inno-combphy.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/phy/rockchip/phy-rockchip-inno-combphy.c b/drivers/phy/rockchip/phy-rockchip-inno-combphy.c
index 62b0d24b5e43..41eaa88d6b98 100644
--- a/drivers/phy/rockchip/phy-rockchip-inno-combphy.c
+++ b/drivers/phy/rockchip/phy-rockchip-inno-combphy.c
@@ -202,8 +202,6 @@ static int phy_pcie_init(struct rockchip_combphy_priv *priv)
 	}
 
 	reset_control_deassert(priv->rsts[PHY_PIPE_RSTN]);
-	/* Release link reset grant */
-	return regmap_write(priv->usb_pcie_grf, 0x0, 0x40004);
 error:
 	return ret;
 }
-- 
2.35.3

